### ISSUE TITLE
svg export crash

### ISSUE THREAD

**Issue by xhoskin** [avocode/avocode#385]
IMPORTANT: Please try to describe how did it happen.

-----
App v.2.4.1-9839e14
Source UID: 24805
```
TypeError: boolean is not a function
  at C:\Users\User\AppData\Local\avocode\app-2.4.1\resources\app.asar\node_modules\@avocode\export-panel\lib\stores\export.js:1:11102
  at C:\Users\User\AppData\Local\avocode\app-2.4.1\resources\app.asar\node_modules\@avocode\core\lib\export.js:1:4967
  at C:\Users\User\AppData\Local\avocode\app-2.4.1\resources\app.asar\node_modules\fstream\node_modules\graceful-fs\graceful-fs.js:104:5
  at FSReqWrap.oncomplete (fs.js:77:15)

```

**Comment by rychlis** [comment]
Thanks for the report, this is a duplicate of #351

Could you please share the design with us at team@avocode.com so we can investigate what happened?

**Comment by petrbrzek** [comment]
Hello, we’ve just released Avocode 2.4.3 that should fix the issue. Let us know if the problem persists.
